.gitignore

```diff
@@ -42,6 +42,7 @@ install_manifest.txt
 *.xcworkspace
 
 # additional *nix generated files
+/ch
```

Comment:
That's my intention. new build script creates a symbolic link for ch on the root. I want it to be ignored on git.